bench/stats.ml

```diff
@@ -0,0 +1,28 @@
+let mean data =
+  let sum = List.fold_left (fun curr_sum b -> curr_sum +. b) 0. data in
+  let n = Int.to_float (List.length data) in
+  sum /. n
+
+let stddev data =
+  let mean = mean data in
+  let sum =
+    List.fold_left
+      (fun curr_sum datapoint ->
+        let squared_diff = Float.pow (datapoint -. mean) 2. in
+        curr_sum +. squared_diff)
+      0. data
+  in
+  let n = Int.to_float (List.length data) in
+  Float.sqrt (sum /. n)
+
+let cmp ?(epsilon=0.01)  a b = 
+  Float.abs (a -. b) < epsilon;;
+
+let sanity_checks () = 
```

Comment:
Useful to have this for sure. Would it be suitable to put this in a test file though?

Reply:
True. Since these are not the usual lock free structure tests, do you think it is more customary to put these in `/test/test_stats.ml` or just `/bench/test_stats.ml`?